content/posts/September-2021.md

```diff
@@ -23,3 +23,13 @@ weight: 71
 - Ogr now supports reacting to the comment with a given reaction, getting them in list and deleting them (only when reaction is added by using ogr API). ([ogr#636](https://github.com/packit/ogr/pull/636))
 - Packit now provides `PACKIT_RPMSPEC_RELEASE` environment variable in actions. ([packit#1363](https://github.com/packit/packit/pull/1363))
 - Dashboard no longer shrinks Copr/Koji icons when being open in narrow view. ([dashboard#140](https://github.com/packit/dashboard/pull/140))
+
+## Week 38 (September 20th - September 24th)
+
+- [Dashboard](dashboard.packit.dev) now shows more readable format of time,
```

Comment:
I like the „sellout“ :)

Reply:
yup! especially for the people who are not familiar with it